dotnet/src/Microsoft.AutoGen/Abstractions/IAgentWorker.cs

```diff
@@ -8,6 +8,7 @@ public interface IAgentWorker
     ValueTask PublishEventAsync(CloudEvent evt, CancellationToken cancellationToken = default);
     ValueTask SendRequestAsync(IAgentBase agent, RpcRequest request, CancellationToken cancellationToken = default);
```

Comment:
The `IAgentRuntime` and `IAgentWorker` looks quite similar, can we combine them?